spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -212,7 +212,7 @@ The `@EnableBinding` annotation can take as parameters one or more interface cla
 
 [NOTE]
 ====
-In Spring Cloud Stream 1.0, the only supported bindable components are the Spring Messaging `MessageChannel` and its extensions `SubscribableChannel` and `PollableChannel`.
+In Spring Cloud Stream 1.0, the only supported bindable components are the Spring Messaging `MessageChannel` and its extension `SubscribableChannel`.
```

Comment:
For the context of this PR, I will update this line as:

In Spring Cloud Stream, the only supported **bindable MessageChannel** components are the Spring Messaging `MessageChannel` and its extension `SubscribableChannel`.`